addons/website_sale/controllers/main.py

```diff
@@ -398,6 +403,7 @@ def shop(self, page=0, category=None, search='', min_price=0.0, max_price=0.0, p
             values['available_max_price'] = tools.float_round(available_max_price, 2)
         if category:
             values['main_object'] = category
+        values.update(self._update_values_hook())
```

Comment:
I'd rather have a function similar to `_get_shop_payment_values`. Though it is more complicated here since we use a lot of local values to create our initial values. So going for something like `_get_additional_shop_values` giving it the initial values seems like a good compromise.
```suggestion
        values.update(self._get_additional_shop_values(values))
```

Reply:
@Williambraecky  thank you for your feedback, applied your suggestion in the code.
Same change for v16:
https://github.com/odoo/odoo/pull/102532

Reply:
I'm closing the 16.0 PR as the forward bot will handle it for us.